nat-lab/Dockerfile

```diff
@@ -5,14 +5,3 @@ FROM ${CONTAINER_REGISTRY}/low-level-hacks/vpn/client/libtelio-build/natlab-deps
 LABEL org.opencontainers.image.authors="info@nordsec.com"
 
 COPY --chmod=0755 bin/ /opt/bin/
-COPY data/nordderper.deb /tmp/
```

Comment:
How does the binary end up in the container? There is no mention of it in Dockerfile.